analysis/Energy_fig4/energy_PI_h1g.m

```diff
@@ -0,0 +1,207 @@
+
+nc64startup;
+
+nc_static  = netcdf('/net2/h1g/CM4_paper/cm4_data/c96/atmos.static.nc', 'nowrite');
+
+nc_PI_TAS = netcdf('/net2/h1g/CM4_paper/cm4_data/PI_C/TAS/tas.nc', 'nowrite');
```

Comment:
@aradhakrishnanGFDL These files are local to a GFDL machine.  Do they need to be modified for this script?  Are there netcdf files somewhere accessible?